examples/step-62/step-62.cc

```diff
@@ -477,9 +477,7 @@ namespace step62
           }
       }
 
-    return std::complex<double>(1,
-                                std::max(calculated_pml_x_coeff,
-                                         calculated_pml_y_coeff));
+    return {1, std::max(calculated_pml_x_coeff, calculated_pml_y_coeff)};
```

Comment:
I don't find this easier to read since I have to scroll way up to see what the return type is. It's a shame that we can use C++14 yet where we could write `1 + std::max(...)*1i`. But maybe we could write `1 + std::max(...)*std::complex<double>(0,1)`?